src/ui/server/controllers/insurance/account/create/your-details/index.ts

```diff
@@ -99,6 +105,12 @@ export const post = async (req: Request, res: Response) => {
   let validationErrors = generateValidationErrors(payload);
 
   if (validationErrors) {
+    // Add SRI to the locals
```

Comment:
could we have a function for this to be dry? can pass it res.locals.SRI

Reply:
Agree, there is already `integrity` middleware (nice) so doing it in the controller I believe is just doing the same thing twice. The middleware will pass `res.locals` to the controller and therefore the rendered template with scripts.

We do this with "get application" every controller that requires an application from the API just consumes `res.locals.application` from `get-application` middleware

Seems we can just remove `res.locals.SRI` from all controller because of the middleware, DRY

Reply:
Sorry currently the middleware doesn't return all SRI's but if it does we wouldn't need such things in the controllers.

Also if we need to add the scripts to other or new pages, with the controller approach, there's a risk that passing SRI could be forgotten, but it'd be done automatically with the middleware